mlir-sycl/include/mlir/Dialect/SYCL/IR/SYCLOps.td

```diff
@@ -42,15 +42,15 @@ class SYCLMethodOpInterfaceImpl<
   list<string> memberFunctionNames = methodNames;
   int arrSize = !size(memberFunctionNames);
 
-  let extraClassDeclaration = [{
+  code extraClassDeclarationBase = [{
```

Comment:
required in order to add more extra class declarations in operators that "implement" this interface.